controllers/duration.js

```diff
@@ -1,9 +1,17 @@
-const log = require('../models/duration');
+const logDuration = require('../models/duration');
 
 module.exports = {
   index,
 };
 
 function index(req, res) {
-  res.send('reps/duration')
-}; 
\ No newline at end of file
+  logDuration.find({}, function (err, duration) {
+    res.render('user/index');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Nlopez4/fitness_tracking_app/pullRequest?prid=7840230)
```suggestion
    res.render("user/index");
```